model/riscv_insts_vext_utils.sail

```diff
@@ -172,6 +172,17 @@ function get_scalar(rs1, SEW) = {
   }
 }
 
+/* Extracts 4 consecutive vector elements starting from index 4*i */
+val get_velem : forall 'n 'm 'p, 'n > 0 & 'm > 0 & 'p >= 0 & 4 * 'p + 3 < 'n. (vector('n, bits('m)), int('p)) -> bits(4 * 'm)
+function get_velem(v, i) = v[4 * i + 3] @ v[4 * i + 2] @ v[4 * i + 1] @ v[4 * i]
```

Comment:
I feel like it's a very generic name for something quite specific. Maybe `get_velem_4` or something?

Reply:
Ah yeah I see. I think it it's only in the Sail snippet it's fine to change it, since eventually we'll want to replace those.

We should use the same definitions at least!